modules/rss/pom.xml

```diff
@@ -28,12 +28,12 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>rome</groupId>
-            <artifactId>rome</artifactId>
-            <version>0.9</version>
+                <groupId>com.rometools</groupId>
+                <artifactId>rome</artifactId>
+                <version>1.5.0</version>
             <exclusions>
                 <exclusion>
-                    <groupId>jdom</groupId>
+                    <groupId>org.jdom</groupId>
```

Comment:
Your answer is contradictory. If you needed to update the dependency because rome needs it, then why are you excluding that dependency here? Am I missing something?